<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QListWidget" name="contentsWidget" >
     <property name="maximumSize" >
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="pagesWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="page" />
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>363</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
